packages/common/src/interfaces.ts

```diff
@@ -83,6 +83,7 @@ export interface StateManagerInterface {
   getProof?(address: Address, storageSlots: Uint8Array[]): Promise<Proof>
   hasStateRoot(root: Uint8Array): Promise<boolean> // only used in client
   shallowCopy(downlevelCaches?: boolean): StateManagerInterface
+  getAppliedKey(address: Uint8Array): Uint8Array
```

Comment:
Ah, this is not possible, since breaking. We need to make this optional for now.

Reply:
Made this optional, and adjusted the code.